src/core/car.py

```diff
@@ -48,19 +58,14 @@ class CarInfo:
 
 @dataclass
 class CarState:
-    __slots__ = "timestamp", "x_velocity", "y_velocity", "location", "angle", "distance"
-
-    timestamp: int
-    """Milliseconds since start of race"""
-
-    x_velocity: float
-    """x velocity (m/s)"""
-    y_velocity: float
-    """y velocity (m/s)"""
-
-    location: Point
-    """Location of the car (x, y)"""
-    angle: float
-    """Angle of car in radians"""
-    distance: float
-    """Distance from start"""
+    __slots__ = "timestamp", "x_velocity", "y_velocity", "location", "angle", "trackDistance"
+
+    timestamp: int      # Milliseconds since race start
+
+    x_velocity: float   # m/s
+    y_velocity: float   # m/s
+
+    location: Point2D   # (x,y)
+    angle: float    # heading angle, radian
+
+    trackDistance: int  # TrackDistance of the Tile it is on
```

Comment:
I like these comments too, but if we use docstrings under the attributes, then language servers like pylance can show the documentation if I hover over it with my mouse.